dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -361,16 +361,17 @@ const DropdownLink = ({ link, index }: DropdownLinkProps) => {
 				))}
 			</a>
 
-			{!!link.notifications?.length && (
-				<div
-					css={css`
-						margin-top: 12px;
-						margin-right: 8px;
-					`}
-				>
-					<NotificationBadge diameter={22} />
-				</div>
-			)}
+			{link.notifications !== undefined &&
+				link.notifications.length > 0 && (
```

Comment:
Once you start to have multiple checks, it might be better to extract its logic to a variable:

```tsx
const hasNotifications: boolean = link.notifications !== undefined && link.notifications.length > 0;

if(hasNotifications) { return <>Notifications!</> }
```